css/features.scss

```diff
@@ -49,7 +43,27 @@
       font-size: 16px;
       font-weight: 400;
       line-height: 30px;
-      color: #3e3e3e
+      color: #3e3e3e;
```

Comment:
You're using SCSS and it allows you to have variables and colors are the best candidate for it.